src/main/java/ca/uwaterloo/swag/pilaipidi/phases/DataFlowAnalyzer.java

```diff
@@ -1,19 +1,11 @@
 package ca.uwaterloo.swag.pilaipidi.phases;
 
-import ca.uwaterloo.swag.pilaipidi.models.ArgumentNamePos;
-import ca.uwaterloo.swag.pilaipidi.models.CFunction;
-import ca.uwaterloo.swag.pilaipidi.models.DFGNode;
-import ca.uwaterloo.swag.pilaipidi.models.DataAccess;
+import ca.uwaterloo.swag.pilaipidi.models.*;
```

Comment:
Let's not do wildcard imports. It will affect runtime performance. Rather, let's import only what is needed.

Reply:
Fixed.